src/components/vr-stereo-toggle/vr-stereo-toggle.js

```diff
@@ -20,23 +20,20 @@ const mapStateToProps = state => ({
   config: state.config.components.vrStereo
 });
 
+const COMPONENT_NAME = 'vrStereo';
```

Comment:
move file to `vr-stereo.js` and also dir name

Reply:
need to fix the `shouldRender` method to get the component config by the new COMPONENT_NAME attribute as per our change of the `disaplyName` attribute - this might be a bug

Reply:
also, all component names start with capital, so please align, same for volume component